src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -1182,6 +1222,18 @@ void Terminal::SetPlayMidiNoteCallback(std::function<void(const int, const int,
     _pfnPlayMidiNote.swap(pfn);
 }
 
+void Terminal::BlinkCursor() noexcept
+{
+    if (_selectionMode != SelectionInteractionMode::Mark)
+    {
+        auto& cursor = _activeBuffer().GetCursor();
+        if (cursor.IsBlinkingAllowed() && cursor.IsVisible())

```

Comment:
This now matches `CursorBlinker.cpp` which doesn't check for `cursor.IsPopupShown()` either (the old code here did).